### PR TITLE
feat: image component

### PR DESCRIPTION
**Pull request by harshitadaswani**
added square image, round image, responsive image

### CODE REVIEW

**Review by bhargaviRdy — APPROVED** [review]
Changes looks good

**Comment by bhargaviRdy** [comment]
> Changes looks good

You can fix this in next PR, the content is hidden while minimising the window and there's no horizontal scroll bar to scroll 
image

File: components/avatar.css [hunk 1]
```diff
@@ -24,7 +24,7 @@
     margin: var(--l-size);
     width: max-content;
```
Comment (JantuDeb) [inline components/avatar.css:25]: You can remove this.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes

**Review by bhargaviRdy — APPROVED** [review]
Looks awesome

**Pull request merged**